### DISCUSSION TITLE
Template Collection

### DISCUSSION THREAD

**Discussion by 0xbismarck** [TriliumNext/Trilium#4567] [General]
Is there a repository or place that people have stored a collection of templates for people to use in their Trilium instance? For instance, while Trilium comes with a Day Template, has anyone ever built a Weekly/Monthly Planner template? 

I see the [book template](https://github.com/zadam/trilium/wiki/Template) comes preinstalled as an example.

**Comment by rodrigoalexandrecosta** [comment]
That would be an interesting source of ideas.

For me, I don't have exactly a monthly/planner template, but I change the Month Note to a Canvas note and use a couple of excalidraw libraries to add (for instance) monthly calendars and habit trackers. I also have templates for books, some specific types of notes (tasks, projects, etc.), some general checklists (e.g. for trips). But for now that it's. I would love to see other use cases that the people over here uses templates to.

**Comment by meichthys** [comment]
Perhaps we could include templates in awesome trillium? @Nriver

**Comment by Nriver** [comment]
Feel free to contribute by adding any valuable template sources you come across to the GitHub repository at https://github.com/Nriver/awesome-trilium. Your contributions are appreciated!

**Comment by meichthys** [comment]
This functionality could potentially be added with #986
